Cargo.toml

```diff
@@ -43,7 +43,7 @@ tracing-limit = { path = "lib/tracing-limit" }
 futures01 = { package = "futures", version = "0.1.25" }
 futures = { version = "0.3", default-features = false, features = ["compat", "io-compat"] }
 tokio01 = { package = "tokio", version = "0.1.22", features = ["io", "tcp", "rt-full", "experimental-tracing", "sync"], default-features = false }
-tokio = { version = "0.2.13", features = ["blocking", "fs", "sync", "macros", "test-util", "rt-core", "io-std"] }
+tokio = { version = "0.2.13", features = ["blocking", "fs", "sync", "macros", "test-util", "rt-core", "io-std", "uds"] }
```

Comment:
Why move `uds` feature from components to dependency?

Reply:
Our `unix` util requires this feature, and without moving it here the `unix` util  would need to be feature guarded by  sources that use it, and in turn they would need to depend on features that `unix` util needs which kinda backwards, so this is a compromise. 

An alternative would be to have separate feature for `unix` util.